File: Task3/task3.go
package main
import (
    "bufio"
    "fmt"
    "os"
)
func main() {
    scanner := bufio.NewScanner(os.Stdin)
    scanner.Scan()
    var numbers int
    fmt.Sscanf(scanner.Text(), "%d", &numbers)
    count := 0
    for i := 0; i < numbers; i++ {
        scanner.Scan()
        var perem int
        fmt.Sscanf(scanner.Text(), "%d", &perem)
        for perem != 0 {
            ost := perem % 10
            if ost%3 == 0 {
                count += ost
            }
            perem /= 10
        }
    }
    fmt.Println(count)
}

File: Task3/task3.java
import java.util.Scanner;
class Main 
{
    public static void main(String[] args)
    {
        Scanner in = new Scanner(System.in);
        int numbers = in.nextInt(); 
        int count = 0;
        while(numbers!=0)
        {
            int perem = in.nextInt(); 
            while (perem != 0) 
            {
                int ost = perem % 10;
                if (ost % 3 == 0)
                {
                    count += ost;
                }
                perem /= 10;
            }
            numbers-=1;
        }
        System.out.println(count);
    }
}
